### PR TITLE
Fix upload batch

### PR DESCRIPTION
**Pull request by wolftk**
- Fix construction of UploadBatch to only include data from the upload from which it was created
- Add GetStatus method to UploadBatch to get the status for a specified file

### CODE REVIEW

File: docs/releaseHistory.md [hunk 1]
```diff
@@ -4,6 +4,11 @@
 
 All releases in the v0.x.x series are subject to breaking changes from one version to another.  After the release of v1.0.0, this project will be subject to [semantic versioning](http://semver.org/).
 
+## v0.0.27
+"Bug Fixes and Enhancements"
```
Comment (kyleburnett) [inline docs/releaseHistory.md]: Can you make this heading consistent with the others (use asterisks to italicize)?

File: proknow-sdk-test/PatientTest/EntitiesTest/EntityScorecardItemTest.cs [hunk 2]
```diff
@@ -145,6 +145,7 @@ public async Task SaveAsyncTest()
                 new MetricBin("FAIL", new byte[] { 255, 0, 0 })
             };
             entityScorecardItem.CustomMetrics = new List<CustomMetricItem>() { customMetricItem2 };
+            //TODO--Modify custom metric value and verify that it is saved
```
Comment (kyleburnett) [inline proknow-sdk-test/PatientTest/EntitiesTest/EntityScorecardItemTest.cs:148]: Did you mean to come back to this before submitting this PR or is this something for the future?

**Review by kyleburnett — CHANGES_REQUESTED** [review]

File: proknow-sdk-test/PatientTest/EntitiesTest/EntityScorecardItemTest.cs (hunk 2 above)
Comment (wolftk) [reply proknow-sdk-test/PatientTest/EntitiesTest/EntityScorecardItemTest.cs:148]: This is something for the future.

File: docs/releaseHistory.md (hunk 1 above)
Comment (wolftk) [reply docs/releaseHistory.md]: Fixed.

**Review by kyleburnett — APPROVED** [review]

**Pull request merged**